### PR TITLE
KEDA release 1.4.0

### PR DESCRIPTION
**Pull request by SatishRanjan**
KEDA release 1.4.0 changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,46 @@
...
+- add deploymentName label if needed ([#709](https://github.com/kedacore/keda/pull/709))
+- Introduce CII Badge ([#691](https://github.com/kedacore/keda/pull/691))
+- Update test dependencies ([#696](https://github.com/kedacore/keda/pull/696))
+- Provide project governance ([#690](https://github.com/kedacore/keda/pull/690))
+- Update zero result return to be non-error inducing ([#695](https://github.com/kedacore/keda/pull/695))
+- Return if kafka offset response is nil ([#689](https://github.com/kedacore/keda/pull/689))
+- Add DCO and remove Microsoft CLA section ([#686](https://github.com/kedacore/keda/pull/686))
+- Fix typos in MySQL scaler ([#683](https://github.com/kedacore/keda/pull/683))
+- Cleanup v1.3 in changelog ([#677](https://github.com/kedacore/keda/pull/677))
+- Update README to mention CNCF ([#682](https://github.com/kedacore/keda/pull/682))
```
Comment (ahmelsayed) [inline CHANGELOG.md:45]: I think the following don't really have any actual changes for a changelog. @tomkerkhove what do you think?

```
 - Provide better guidance around contributions & DCO ([#720](https://github.com/kedacore/keda/pull/720))
 - Remove mit license ([#716](https://github.com/kedacore/keda/pull/716))
 - Link to guide for adding new scaler docs ([#715](https://github.com/kedacore/keda/pull/715))
 - Correct deploying methods ([#714](https://github.com/kedacore/keda/pull/714))
 - Introduce CII Badge ([#691](https://github.com/kedacore/keda/pull/691))
 - Update test dependencies ([#696](https://github.com/kedacore/keda/pull/696))
 - Provide project governance ([#690](https://github.com/kedacore/keda/pull/690))
 - Add DCO and remove Microsoft CLA section ([#686](https://github.com/kedacore/keda/pull/686))
 - Cleanup v1.3 in changelog ([#677](https://github.com/kedacore/keda/pull/677))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,37 @@
...
+## v1.4
+
+### New
+
+- Extend RabbitMQ scaler to support count unacked messages([#700](https://github.com/kedacore/keda/pull/700))
+
+### Improvements
+
+- Fix scalers leaking ([#684](https://github.com/kedacore/keda/pull/684))
+- Provide installation YAML in the release ([#740](https://github.com/kedacore/keda/pull/740))
```
Comment (tomkerkhove) [inline CHANGELOG.md:16]: Maybe this is more clear what it means?
```suggestion
- Provide installation YAML package as release artifact ([#740](https://github.com/kedacore/keda/pull/740))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,37 @@
...
+- Handle nil pointers and empty arrays properly in Azure Monitor Scaler ([#680](https://github.com/kedacore/keda/pull/680))
```
Comment (tomkerkhove) [inline CHANGELOG.md:17]: Maybe this is more clear from a function standpoint? Feels a bit too technical for a changelog.
```suggestion
- Improve Azure Monitor scaler to handle queries without metrics ([#680](https://github.com/kedacore/keda/pull/680))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,37 @@
...
+- Authenticate to AWS with dedicated role without AssumeRole permissions ([#656](https://github.com/kedacore/keda/pull/656))
+
+### Breaking Changes
+
+None.
+
+### Other
+
+- Adding label for metrics service selection ([#745](https://github.com/kedacore/keda/pull/745))
+- Provide installation YAML in the release ([#740](https://github.com/kedacore/keda/pull/740))
```
Comment (tomkerkhove) [inline CHANGELOG.md:27]: Already mentioned above
```suggestion
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,37 @@
...
+
+- Extend RabbitMQ scaler to support count unacked messages([#700](https://github.com/kedacore/keda/pull/700))
+
+### Improvements
+
+- Fix scalers leaking ([#684](https://github.com/kedacore/keda/pull/684))
+- Provide installation YAML in the release ([#740](https://github.com/kedacore/keda/pull/740))
+- Handle nil pointers and empty arrays properly in Azure Monitor Scaler ([#680](https://github.com/kedacore/keda/pull/680))
+- Authenticate to AWS with dedicated role without AssumeRole permissions ([#656](https://github.com/kedacore/keda/pull/656))
+
```
Comment (tomkerkhove) [inline CHANGELOG.md:19]:
```suggestion
- KEDA now respects label restrictions on Horizontal Pod Autoscaler to have max 63 chars ([#707](https://github.com/kedacore/keda/pull/707))
- KEDA will automatically assign `deploymentName` label if it was not defined in `ScaledObject` ([#709](https://github.com/kedacore/keda/pull/709))

```

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,37 @@
...
+None.
+
+### Other
+
+- Adding label for metrics service selection ([#745](https://github.com/kedacore/keda/pull/745))
+- Provide installation YAML in the release ([#740](https://github.com/kedacore/keda/pull/740))
+- Filter returned metrics from api server based on queried name ([#732](https://github.com/kedacore/keda/pull/732))
+- Add redis host and port parameter to the scaler with tests ([#719](https://github.com/kedacore/keda/pull/719))
+- Remove go micro version ([#718](https://github.com/kedacore/keda/pull/718))
+- HPA: label can have max 63 chars ([#707](https://github.com/kedacore/keda/pull/707))
```
Comment (tomkerkhove) [inline CHANGELOG.md:31]: I'd say this is an improvement
```suggestion
```

See above

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,37 @@
...
+- add deploymentName label if needed ([#709](https://github.com/kedacore/keda/pull/709))
```
Comment (tomkerkhove) [inline CHANGELOG.md:32]: Moved to improvement as well
```suggestion
```

**Comment by tomkerkhove** [comment]
Don't forget about our new step 5 in https://github.com/kedacore/keda/blob/master/RELEASE-PROCESS.MD

**Comment by zroubalik** [comment]
Good comments @tomkerkhove !

**Pull request merged**